### PR TITLE
Add Readiu Pella

### PR DESCRIPTION
**Pull request by readiu**
Please review. 
Starting the Pella file :8ball: 

Please provide feedback on my town.
@jeremiahmegel  any thing you want to add?
Need a :hamster: 
how about :hankey:

### CODE REVIEW

File: Readiu-Pella.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Pella
+### To Do
+
```
Comment (jeremiahmegel) [inline Readiu-Pella.md]: You mean there's nothing to do in Pella?

File: Readiu-Pella.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Pella
```
Comment (aharden) [inline Readiu-Pella.md]: Where is Pella?

Comment (readiu) [inline Readiu-Pella.md]: Iowa!

File: Readiu-Pella.md (hunk 1 above)
Comment (readiu) [inline Readiu-Pella.md]: Yep!

Comment (readiu) [inline Readiu-Pella.md]: If you want to watch old people in wooden shoes, go for it!! 
Or you can go jump in a lake LOL :smiley_cat:

File: Readiu-Pella.md (hunk 2 above)
Comment (aharden) [inline Readiu-Pella.md]: Never been to Iowa - I would change the first line to:

    Pella, IA

Lake Red Rock looks pretty cool!

File: Readiu-Pella.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+### To Do
+
+
+### To See
```
Comment (readiu) [inline Readiu-Pella.md]: Pella, IA

File: Readiu-Pella.md (hunk 2 above)
Comment (jeremiahmegel) [inline Readiu-Pella.md]: >Oh, this is the Pella, the home of Pella Windows and Doors! :door:

Yessir!

**Comment by jeremiahmegel** [comment]
:shipit:!

**Comment by aharden** [comment]
🚢

**Pull request merged**